src/main/scripts/src/app/shared/const.ts

```diff
@@ -0,0 +1,80 @@
+export const placeHolderNewCollection = 'Collection name';
```

Comment:
I think this does not need to be here. It will be one of the localized strings in the feature and almost certainly put somewhere else. So for now, I think can stay directly inside the component where it is needed.